.github/workflows/pack-build_and_test.yaml

```diff
@@ -14,7 +14,7 @@ on:
       python-version:
         required: false
         type: string
-        default: "3.6"
+        default: ["3.6","3.8","3.9"]
```

Comment:
Changing the type from string to array here would mean we have to change the matrix definition below. There, we are using `matrix.include` to add one entry for the indicated python version.

The docs about `matrix.include` are here.: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations